modules/boltlib/lib/puppet/functions/run_command.rb

```diff
@@ -5,18 +5,22 @@
 # * A target is a String with a targets's hostname or a Target.
 # * The returned value contains information about the result per target.
 #
+require 'bolt/error'
+
 Puppet::Functions.create_function(:run_command) do
   local_types do
     type 'TargetOrTargets = Variant[String[1], Target, Array[TargetOrTargets]]'
   end
 
   dispatch :run_command do
     param 'String[1]', :command
-    repeated_param 'TargetOrTargets', :targets
+    param 'TargetOrTargets', :targets
```

Comment:
Ok, these changes all are more consistent with run_task. That makes sense.

Reply:
yeah that was my thinking. They were kind of forced since we can't accept both splatted options and targets.